lib/solarwinds_apm/opentelemetry/solarwinds_response_propagator.rb

```diff
@@ -56,31 +56,20 @@ def inject(carrier, context: ::OpenTelemetry::Context.current,
                        xtraceoptions_response)
           end
           setter.set(carrier, HTTP_HEADER_ACCESS_CONTROL_EXPOSE_HEADERS, exposed_headers.join(','))
-        end
-
-        # Returns the predefined propagation fields. If your carrier is reused, you
-        # should delete the fields returned by this method before calling +inject+.
-        #
-        # @return [Array<String>] a list of fields that will be used by this propagator.
-        def fields
-          TRACESTATE_HEADER_NAME
+        rescue StandardError => e
+          SolarWindsAPM.logger.debug { "[#{self.class}/#{__method__}] Injection failed: #{e.message}" }
         end
 
         private
 
-        # sw_xtraceoptions_response_key -> xtrace_options_response
```

Comment:
Nit: I think it would still be good to have comment/docstring for this function

Reply:
Added back